### ISSUE TITLE
Learning Based Feature Extractor Supplementary Material

### ISSUE THREAD

**Issue by shannon112** [shannon112/awesome-ros-mobile-robot#11]
🔅 Learning-Based Feature Extractor  
 * "Alexnet" 
   > Krizhevsky, Alex. "One weird trick for parallelizing convolutional neural networks." arXiv preprint arXiv:1404.5997 (2014).
 * "VGG" 
   > Simonyan, Karen, et al. "Very deep convolutional networks for large-scale image recognition." arXiv preprint arXiv:1409.1556 (2014).
 * "ResNet" 
   > He, Kaiming, et al. "Deep residual learning for image recognition." Proceedings of the IEEE conference on computer vision and pattern recognition. 2016.
 * "InceptionV3"
   > Szegedy, Christian, et al. "Rethinking the inception architecture for computer vision." Proceedings of the IEEE conference on computer vision and pattern recognition. 2016.
 * "DenseNet"
   > Huang, Gao, et al. "Densely connected convolutional networks." Proceedings of the IEEE conference on computer vision and pattern recognition. 2017.
 * "GoogleNet"
   > Szegedy, Christian, et al. "Going deeper with convolutions." Proceedings of the IEEE conference on computer vision and pattern recognition. 2015.
 * "MobileNet"
   > Howard, Andrew G., et al. "Mobilenets: Efficient convolutional neural networks for mobile vision applications." arXiv preprint arXiv:1704.04861 (2017).
 * "SqueezeNet"
   > Iandola, Forrest N., et al. "SqueezeNet: AlexNet-level accuracy with 50x fewer parameters and< 0.5 MB model size." arXiv preprint arXiv:1602.07360 (2016).
 * Pytorch implementation: https://pytorch.org/docs/stable/torchvision/models.html
